san/sanbase_graphql.py

```diff
@@ -106,6 +105,20 @@
     'projects': {
         'query': 'allProjects',
         'return_fields': ['name', 'slug', 'ticker', 'totalSupply', 'marketSegment']
+    },
+    'eth_top_transactions': {
+        'query': 'ethTopTransactions',
+        'return_fields': [
+            'datetime',
+            {
+                'fromAddress': ['address', 'isExchange']
```

Comment:
This should always be a map with a single key-value pair. In this case, this could be a tuple instead?
```python
'return_fields': [
'datetime',
('fromAddress', ['address', 'isExchange']),
...
```